macro/Cargo.toml

```diff
@@ -16,6 +16,7 @@ description = "Generate types and helpers for interacting with Substrate runtime
 [features]
 web = ["subxt-codegen/web"]
 runtime-path = ["polkadot-sdk"]
+runtime-metadata-insecure-url = ["subxt-utils/fetch-metadata-url"]
```

Comment:
Nit: Just to align with the above I'd clal it `runtime-url` orrr, rename both to match the attrs so we have:

`runtime-metadata-path`
`runtime-metadata-insecure-url`